Task_Position_the_Cubes/index.css

```diff
@@ -1 +1,72 @@
-/*  Start coding below for POSITION THE CUBES TASK*/
+/*  Start coding below for POSITION THE CUBES TASK*/

```

Comment:
This line has a syntax error. There is an extra asterisk (*) that should be removed.